auem/data/datasets/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .audioset import AudiosetAnnotationReaderV1, AudiosetDataset
```

Comment:
Why is this V1 here, but we're using V2 below? copypasta?

Reply:
I'm still using it; it's nice to have descriptions and examples for each class!